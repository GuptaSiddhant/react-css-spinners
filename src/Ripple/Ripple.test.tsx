import React from 'react'
import renderer from 'react-test-renderer'
import { Ripple } from './'

it('renders correctly', (): void => {
  const tree = renderer.create(<Ripple />).toJSON()
  expect(tree).toMatchSnapshot()
})
