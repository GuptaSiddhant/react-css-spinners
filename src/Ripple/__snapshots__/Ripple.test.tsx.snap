// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="rcs-ripple"
  style={
    Object {
      "--rcs-ripple-color": "#fff",
      "--rcs-ripple-size": "64px",
      "--rcs-ripple-thickness": "4px",
    }
  }
>
  <div />
  <div />
</div>
`;
